lib/scrivener/html.ex

```diff
@@ -240,11 +254,13 @@ defmodule Scrivener.HTML do
     add_first(paginator.page_number, options[:distance], options[:first])
     |> add_previous(paginator.page_number)
     |> page_number_list(paginator.page_number, paginator.total_pages, options[:distance])
+    |> add_ellipsis(paginator.page_number, paginator.total_pages, options[:distance], options[:ellipsis])
     |> add_last(paginator.page_number, paginator.total_pages, options[:distance], options[:last])
     |> add_next(paginator.page_number, paginator.total_pages)
     |> Enum.map(fn
       :next -> if options[:next], do: {options[:next], paginator.page_number + 1}
       :previous -> if options[:previous], do: {options[:previous], paginator.page_number - 1}
+      :ellipsis -> if options[:ellipsis], do: {:ellipsis, paginator.page_number + 1}
```

Comment:
page number doesn't make sense here... I think we should return `{:ellipsis, options[:ellipsis]}`, which would be the text to show.